src/utils/Membership.js

```diff
@@ -5,6 +5,13 @@ const onKeyPress = event => {
   if (/\+|-|e/.test(keyValue)) event.preventDefault();
 };
 
+const amountFormatter = amount => {
```

Comment:
Tangentially related to the work here, but initially found this confusing because this is not strictly speaking `Membership` specific logic. Obviously, small quibble but maybe this function belongs elsewhere or the filename should be changed.

Reply:
Yeah, I guess these shared util functions should be broken-out into something more generic-sounding; will tidy this up in another PR if that's cool